src/js/hooks/cycleCount/useToCountTab.jsx

```diff
@@ -176,19 +176,27 @@ const useToCountTab = ({
     }),
     columnHelper.accessor('internalLocations', {
       header: () => (
-        <TableHeaderCell sortable columnId="internalLocations" {...sortableProps}>
+        <TableHeaderCell>
           {translate('react.cycleCount.table.binLocation.label', 'Bin Location')}
         </TableHeaderCell>
       ),
-      cell: ({ getValue }) => (
-        <TableCell
-          className="rt-td"
-          tooltip
-          tooltipLabel={getValue()}
-        >
-          {mapStringToList(getValue(), ',', 100).map((binLocationName) => <div>{binLocationName}</div>)}
-        </TableCell>
-      ),
+      cell: ({ getValue }) => {
+        const binLocationList = mapStringToList(getValue(), ',');
```

Comment:
How long can this list potentially be?

Reply:
The longest list that I have locally is 13 bin locations

Reply:
Should it be capped at 4 bins like on the other page?

Reply:
it is, but in OBPIH-6942